__tests__/app.test.js

```diff
@@ -0,0 +1,45 @@
+const testData = require("../db/data/test-data/index");
+const app = require("../app.js");
+const seed = require("../db/seeds/seed.js");
+const db = require("../db/connection.js");
+const request = require("supertest");
+
+//what do we want out of our endpoints??
+
+beforeEach(() => {
+  return seed(testData);
+});
+afterAll(() => {
+  return db.end();
+});
+
+describe("NC News API testing", () => {
+  describe("GET /api/topics - will get all topics in the database", () => {
+    test("/api/topics responds with a 200 status code", () => {
```

Comment:
we don't need this test, as the one below is showing that a 200 comes back. I would just amend the test description below to also say "status 200" at the start 🙂